### ISSUE TITLE
Add feature to download test-generated artifacts

### ISSUE THREAD

**Issue by mmcc007** [mmcc007/sylph#22]
Flutter tests can generate artifacts such as screenshots and logs. Add these artifacts to the job artifacts to be included in download.

**Comment by vishna** [comment]
Yes, please. I'm using [ozzie](https://github.com/bmw-tech/ozzie.flutter) to add reports to my tests and it would be nice to just download that folder somehow from AWS.

**Comment by mmcc007** [comment]
It looks like this would also be required for #64

**Comment by fvisticot** [comment]
any progress on this feature ?
